docs/GraFx-Developers/connectors/data-connector/data-connector-fundamentals/index.md

```
@@ -125,6 +125,19 @@ The returned `DataModel` structure is used to define the types of data that your
 - `multiLine`: For multi-line text
 - `date`: For date values
 
+!!! Warning "Template variable mapping"
+    Data returned by a Data Connector can be mapped to template variables. Most supported schema types map 1:1, with a few exceptions.
+
+    There are no dedicated `image` or `list` types in the supported data model. To map data to an `Image Variable` or a `List Variable`, return that field as `singleLine`.
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)